Launch/ParakeetProE.launch

```diff
@@ -0,0 +1,19 @@
+<launch>
+	<param name="sensorType" value="ProE" />
+	<param name="ipAddress" value="192.168.158.98" />
+	<param name="dstPort" value="6543" />
+	<param name="srcPort" value="6668" />
+	<param name="intensityData" value="true" />
+	<param name="scanningFrequency_Hz" value="15" />
+	<param name="dataSmoothing" value="false" />
```

Comment:
I think we should default the dataSmoothing and dragPointRemoval to true.

Reply:
I'm fine doing this. Just for the record, Locus did not modify these. Let me know if you still want me to go through with this change.

Reply:
If they left it disabled let's leave it as-is.